ArduCopter/ekf_check.cpp

```diff
@@ -264,14 +275,11 @@ void Copter::check_vibration()
     const bool innov_velD_posD_positive = is_positive(vel_innovation.z) && is_positive(pos_innovation.z);
 
     // check if vertical velocity variance is at least 1 (NK4.SV >= 1.0)
-    float position_variance, vel_variance, height_variance, tas_variance;
-    Vector3f mag_variance;
-    if (!ahrs.get_variances(vel_variance, position_variance, height_variance, mag_variance, tas_variance)) {
+    if (!variances_valid) {
```

Comment:
maybe we should add a comment here that the filtered variances are updated in ekf_check() just so that the dependency is clear.

Reply:
Fixed